### DISCUSSION TITLE
is it possible to select candidates cyclically ?

### DISCUSSION THREAD

**Discussion by bo-wu** [ibhagwan/fzf-lua#1379] [Q&A] [answered]
As the title, how can I select search candidates from the first one to the last one inversely? By far, I have to move from the start to end either by ctrl+n/j or ctrl+d . If there are too many candidates, I have to move several times.

**Comment by ibhagwan** [comment]
Not sure if that’s what you mean but you can toggle all with `alt-a` by default:
https://github.com/ibhagwan/fzf-lua/blob/73bdec9ac5da578376bdc5a705ea80a19baa4942/lua/fzf-lua/defaults.lua#L110

For other selections read `man fzf` as this behavior is contellled by the binary fzf.

**Reply by bo-wu** [reply]
Thanks for reply. I read manual of fzf. I think 'fzf --cycle' is what I need.  So now I have to figure out how to add the parameter to the command in fzf-lua. Would you please give me some clue?

Thanks in advance

**Reply by ibhagwan** [reply, marked as answer]
See #1202

**Reply by bo-wu** [reply]
Thanks a lot. I manage to add the option in my plugin config.
